src/LRUCache.jl

```diff
@@ -232,4 +233,23 @@ function _finalize_evictions!(finalizer, evictions)
     return
 end
 
+# Reverse iterator for LRUCache.LRU
+Base.eltype(::Type{Iterators.Reverse{LRUCache.LRU{T}}}) where {T} = eltype(T)
+Base.IteratorSize(::Type{Iterators.Reverse{LRUCache.LRU{T}}}) where {T} = Base.IteratorSize(LRUCache.CyclicOrderedSet)
+Base.IteratorEltype(::Type{Iterators.Reverse{LRUCache.LRU{T}}}) where {T} = Base.IteratorSize(LRUCache.LRU)
+Base.length(lru::Iterators.Reverse{LRUCache.LRU{T}}) where {T} = length(lru.keyset)
```

Comment:
`lru` is the `Reverse` object, so its field should probably be `lru.itr`. Did you test this?

Reply:
Yes it was tested. See the unit tests added.

Reply:
I'll fix this. I agree it should be `lru.itr.keyset`